packages/cli/src/commands/base-command.ts

```diff
@@ -75,7 +75,11 @@ export abstract class BaseCommand extends Command {
 	protected async loadModules() {
 		for (const moduleName of this.modulesConfig.modules) {
 			// add module to the registry for dependency injection
-			await import(`../modules/${moduleName}/${moduleName}.module`);
+			try {
+				await import(`../modules/${moduleName}/${moduleName}.module`);
+			} catch {
+				await import(`../modules/${moduleName}/${moduleName.replace('.ee', '')}.module.ee`);
```

Comment:
Are adding the suffix to the module name, like `external-secrets.ee`? 

How about we're just dropping the suffix?